tests/suite/utils/resources_utils.py

```diff
@@ -1734,3 +1734,36 @@ def get_resource_metrics(kube_apis, plural, namespace="nginx-ingress") -> str:
     else:
         return "Invalid plural specified. Please use 'pods' or 'nodes' as the plural"
     return metrics["items"]
+
+
+def assert_pods_scaled_to_count(apps_v1_api, v1, deployment_name, namespace, expected_count, timeout=60, interval=1):
```

Comment:
@j1m-ryan Can this method be moved to https://github.com/nginxinc/kubernetes-ingress/blob/fix/use-cluster-ip-without-reload/tests/suite/utils/custom_assertions.py

Reply:
yes will do